pkg/components/dex/component_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dex //nolint:testpackage
```

Comment:
Please try using only exported API in tests if possible.
```suggestion
package dex_test
```

Reply:
But we need `newComponent`

Reply:
Nope, you can use `components.GetComponent()` :)